.github/workflows/deploy.yml

```
@@ -0,0 +1,99 @@
+name: CI/CD Frontend
+
+on:
+  push:
+    branches: [ "deploy" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)